DuoSecurity.Auth.Http/JsonModels/DeviceModel.cs

```diff
@@ -8,7 +8,19 @@ namespace DuoSecurity.Auth.Http.JsonModels
 {
     internal class DeviceModel
     {
-        public IEnumerable<string> Capabilities { get; set; }
+        private IEnumerable<string> _capabilities;
+
+        public IEnumerable<string> Capabilities
+        {
+            get
+            {
+                if (_capabilities == null)
+                    _capabilities = new List<string>();
+
+                return _capabilities;
+            }
+            set { _capabilities = value; }
```

Comment:
I tend to prefer that a backing field check occurs on the `set` rather than on the `get`, since in my mind the `set` would be less frequent - so I would invert this so the `get` just returns the backing field while the `set` checks if the incoming `value == null` and handles that accordingly.